python/sqlalchemy_openmldb/openmldb_dialet.py

```diff
@@ -86,9 +89,9 @@ def __init__(self, dialect):
     def format_drill_table(self, schema, isFile=True):
         pass
 
-class OpenMLDBDialect(default.DefaultDialect):
+class OpenmldbDialect(default.DefaultDialect):
     
-    name = "openmldbapi"
+    name = "openmldb_dialet"
```

Comment:
openmldb_dialet -> openmldb_dialect
also, the filename should be `openmldb_dialect.py`